lib/ui/widgets/icon_row.dart

```diff
@@ -7,12 +7,14 @@ class IconRow extends StatelessWidget {
   final double iconSize;
   final Iterable<IconWithLabel> icons;
   final MainAxisAlignment mainAxisAlignment;
+  final void Function({required String query})? onIconPressed;
```

Comment:
**suggestion:** Rename parameter from query to value or selected... since this widget could be use in another context the query would not make sense

Reply:
can not rename without crashing pls help